tools/docker/builder/Dockerfile

```diff
@@ -135,6 +139,15 @@ RUN set -xe; \
     echo "${JQ_SHA256}  jq-${JQ_OS}" | sha256sum -c; \
     mv jq-${JQ_OS} /usr/bin/jq; \
     chmod +x /usr/bin/jq; \
+# Install for release
+    cargo install --locked git-cliff --version 0.7.0; \
+    cargo install --locked tomlq --version 0.1.0; \
+    cargo install --locked cargo-release --version 0.21.0; \
```

Comment:
Thank you for pinning the versions 👍🏻 

 I'm familiar with git-cliff already. Is `tomlq` a runtime dependency of `cargo-readme` in some way? Or already included in other release steps?

Reply:
tomlq is used in other release steps, we use it to get version of crates during release.